<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CapturaNeurona</class>
 <widget class="QDialog" name="CapturaNeurona">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditVoltaje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltaje</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPosicion_x">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Posicion_x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPosicion_y">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Posicion_y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGreen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBlue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; font-style:italic;&quot;&gt;Ingresa los datos correspondientes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGuardar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
